packages/siop-oid4vp/lib/types/SIOP.types.ts

```diff
@@ -179,6 +179,7 @@ export interface AuthorizationResponsePayload {
     | MdocOid4vpMdocVpToken
   presentation_submission?: PresentationSubmission
   verifiedData?: IPresentation | AdditionalClaims
+  is_first_party?: boolean
```

Comment:
not related, but each type has a different format between snake_case and camelCase